ReactiveCocoa/Swift/Lifetime.swift

```diff
@@ -27,13 +47,13 @@ extension NSObject {
 		objc_sync_enter(self)
 		defer { objc_sync_exit(self) }
 
-		if let lifetime = objc_getAssociatedObject(self, &lifetimeKey) as! Lifetime? {
-			return lifetime
+		if let token = objc_getAssociatedObject(self, &lifetimeKey) as! Lifetime.Token? {
+			return Lifetime(token)
```

Comment:
Any reason not to store the `Lifetime`, to avoid allocating a different one each time?